imagekit/conf.py

```diff
@@ -17,25 +17,17 @@ class ImageKitConf(AppConf):
 
     def configure_cache_backend(self, value):
         if value is None:
-            try:
-                from django.core.cache.backends.dummy import DummyCache
-            except ImportError:
-                dummy_cache = 'dummy://'
-            else:
-                dummy_cache = 'django.core.cache.backends.dummy.DummyCache'
-
             # DEFAULT_CACHE_ALIAS doesn't exist in Django<=1.2
             try:
-                from django.core.cache import DEFAULT_CACHE_ALIAS as default_cache_alias
+                from django.core.cache import DEFAULT_CACHE_ALIAS
+                default_cache_alias = DEFAULT_CACHE_ALIAS or 'default'
```

Comment:
Thanks @vstoykov for the explanation, very clear. I will restore the behavior then